docs/troubleshooting-guide/cant-see-tables.md

```diff
@@ -0,0 +1,85 @@
+
+## Does the table exist and can the Metabase account access it?
+
+**Root cause:** The table you think you should be able to see does not exist (e.g., it has a different name than you expect), or the login ID that Metabase uses to query the database does not have privileges to view the table.
+
+**Steps to take:** To test that the table you are trying to query actually exists and that you have permission to access it, use the SQL Editor to create and run a query like:
+
+```
+select * from SOMEWHERE
+```
+
+where `SOMEWHERE` is the table you think you should be able to see. Metabase should display an error message like:
+
+```
+Table "SOMEWHERE" not found
+```
+
+If you see this message, use another application (e.g., `psql` for PostreSQL) to send the same query to the database. If it also produces a "table not found" message, check the database schema and the spelling of the table name.
+
+Be sure to log in using the same credentials that Metabase uses. A common source of problems is that the Metabase "user" does not have the same privileges as a member of IT staff or a developer, so tables that are visible to the latter using external applications are not visible to Metabase.
+
+## Does the person who cannot see the table have permission to view it?
+
+*Root cause:** Metabase uses a group-based permission model: people belong to groups, and administrators can set permissions so that some groups cannot see all of the tables. (It also allows administrators to control which rows or columns specific people can see---issues with that are covered in the troubleshooting guide for [sandboxing][sandboxing].)
+
+**Steps to take:**
+
+1. Log into Metabase using the ID of the person who cannot see the expected tables.
+2. Confirm that the tables are not visible.
+3. Log out, then log in using the administrator's credentials.
+
+If the administrator's account can see the tables but an individual person cannot:
+
+1. Go to Admin > Permissions and see if any groups have been denied access to the table.
+2. If any groups have been denied access, go to Admin > People and look at the "Groups" column for the person who cannot see the expected tables. If they are in a group that does not have access to the table, you may need to move them to another group or change table permissions.
+
+## Is Metabase's metadata out of sync with the state of the database?
+
+**Root cause:** In order to display available tables and columns in dropdown menus and previews, Metabase runs a query every hour to find out what tables are available and what columns are in each available table, and stores this information in its application database.
+
+1. If a table has been added or removed since the last time this "sync" operation ran, Metabase's information about the database will be incorrect.
```

Comment:
```suggestion
1. If a table has been added or removed since the last time this "sync" operation ran, Metabase's information about the database will be outdated.
```